src/preset.ts

```diff
@@ -453,7 +453,7 @@ export function setCompilers(_kits: Kit[]) {
 function merge<T extends Object>(target: T, base: T) {
     Object.keys(base).forEach(key => {
         const field = key as keyof T;
-        if (!target[field]) {
+        if (!target.hasOwnProperty(field)) {
```

Comment:
better to use Object.hasOwn?

Reply:
Fair point, though I don't think target will be `null`.  I can change it anyway.

Reply:
Hmm... IntelliSense doesn't offer that API, so it might not be available in VS Code yet. It looks like browsers only just got it a few months ago.

Reply:
Then please use Object.prototype.hasOwnProperty.call  instead